packages/flutter_arc_text/lib/src/arc_text.dart

```diff
@@ -95,27 +95,39 @@ class ArcText extends StatelessWidget {
   final PainterDelegate painterDelegate;
 
   @override
-  Widget build(BuildContext context) => CustomPaint(
-        painter: _Painter(
-          radius: radius,
-          text: text,
-          textStyle: textStyle,
-          alignment: startAngleAlignment,
-          initialAngle: startAngle,
-          direction: direction,
-          placement: placement,
-          stretchAngle: stretchAngle,
-          interLetterAngle: interLetterAngle,
-          painterDelegate: painterDelegate,
-        ),
-      );
+  Widget build(BuildContext context) {
+    final DefaultTextStyle defaultTextStyle = DefaultTextStyle.of(context);
+    TextStyle? effectiveTextStyle = textStyle;
+    if (textStyle == null || textStyle!.inherit) {
```

Comment:
I don't think this check is needed, `merge` takes into account the `inherit` property of the parameter.

Reply:
ack, looking at merge(..) you are correct. This is just what Text does, but I
suspect merge just changed underneath them. I'll fix.